stories/topics/proc-saas-set-up-public.adoc

```diff
@@ -24,6 +26,9 @@ As an AWS customer you can search for and subscribe to {SaaSonAWS} from the AWS
 .. Link your AWS account to the Red{nbsp}Hat account.
 . This then directs you to the *Provision environment* page on the Red{nbsp}Hat Hybrid Cloud Console. Here you can start configuring your environment. 
 . Enter your AWS account ID.
++
+NOTE: This account ID must be the account ID that purchased the offer from the AWS marketplace. 
```

Comment:
```suggestion
This account ID must be the account ID that purchased the offer from the AWS marketplace. 
```
Suggestion: omit Note, as this is a reminder of something that was mentioned already